### ISSUE TITLE
The whole screen goes black when first time entering it or refresh it afterwards.

### ISSUE THREAD

**Issue by hacker0limbo** [playcode/playcode#32]
Hello, just wanna report a small bug?(may be)

![bug](https://user-images.githubusercontent.com/36297538/73596924-df605c80-4561-11ea-9048-5eefd4586734.gif)

As you can see from the gif, when the first time entering into it, it starts to render normally then whole screen becomes blank, and wait about 0.5s it continues to render and show the editor screen.

This issue happens when you refresh the page as well.

Basically it does not cause any editoring issue, its just for better UI performance if this bug could be fixed, for example during the rendering, adding a loading indication would be a solution?

Cheers!

**Comment by ianberdin** [comment]
Hi, I’m Ruslan.
Thank you for your feedback!	
Yep, this is strange… I will fix this. Yes, spinner is suitable for this!

Ruslan Ianberdin,
Founder of playcode.io

**Comment by hacker0limbo** [comment]
Cool, will close this issue then.
